impl/src/main/java/org/jboss/cdi/tck/interceptors/tests/order/aroundInvoke/Interceptor2.java

```diff
@@ -18,14 +18,16 @@
 
 import static org.testng.Assert.assertEquals;
 
+import jakarta.annotation.Priority;
 import jakarta.interceptor.AroundInvoke;
+import jakarta.interceptor.Interceptor;
```

Comment:
These 2 new imports are unused? (The interceptor isn't enabled, but that seems to be on purpose?)